src/core/00_vnet.tf

```diff
@@ -12,12 +13,52 @@ module "vnet" {
   tags          = var.tags
 }
 
+# pvt endpoint snet on primary
 module "private_endpoint_snet" {
   source                                    = "./.terraform/modules/__v3__/subnet/"
   name                                      = format("%s-%s-snet", local.project, "private_endpoint")
   address_prefixes                          = var.cidr_pvt_endp_snet
   resource_group_name                       = azurerm_resource_group.networking.name
   virtual_network_name                      = module.vnet.name
   private_endpoint_network_policies_enabled = false
-  service_endpoints                         = ["Microsoft.Sql"]
+  service_endpoints                         = []
+}
+
+# secondary vnet
+module "secondary_vnet" {
+  source              = "./.terraform/modules/__v3__/virtual_network/"
+  name                = format("%s-secondary-vnet", local.project)
+  location            = azurerm_resource_group.secondary_networking.location
+  resource_group_name = azurerm_resource_group.secondary_networking.name
+  # FIXME, move to variables
+  ddos_protection_plan = {
+    id     = "/subscriptions/0da48c97-355f-4050-a520-f11a18b8be90/resourceGroups/sec-p-ddos/providers/Microsoft.Network/ddosProtectionPlans/sec-p-ddos-protection"
+    enable = true
+  }
+  address_space = var.secondary_cidr_vnet
+  tags          = var.tags
+}
+
+# pvt endpoint snet on secondary
+module "private_endpoint_secondary_snet" {
+  source                                    = "./.terraform/modules/__v3__/subnet/"
+  name                                      = format("%s-%s-snet", local.project, "secondary_private_endpoint")
```

Comment:
```suggestion
  name                                      = format("%s-%s-snet", local.project, "private_endpoint")
```

Reply:
we can reuse private_endpoint, because is another vnet, only a suggestion not blocking